test/plugin/scenarios/servicecomb-1.x-scenario/src/main/java/org/apache/skywalking/apm/testcase/servicecomb/provider/CodeFirstJaxrsHelloImpl.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.testcase.servicecomb.provider;
+
+
+import org.apache.servicecomb.provider.pojo.RpcSchema;
```

Comment:
The package name has been changed, I think we need a new plugin?

Reply:
Already provided [servicecomb-plugin](https://github.com/apache/skywalking/tree/master/apm-sniffer/apm-sdk-plugin/servicecomb-plugin)

Reply:
I see.